Example/Example/Common/BankCard/BankCardViewController.swift

```diff
@@ -75,8 +129,63 @@ private extension BankCardViewController {
 
 private extension BankCardViewController {
 
-    func bindTextInputChangesHandler() {
-        // TODO: Implement.
+    func bindTextInputEventHandlers() {
+        cardNumberTextInputBinding.eventHandler = { [unowned self] event in
+            switch event {
+            case let .editingChanged(state, changes):
```

Comment:
Because of checking changes multiple times in many places why don't you want to add optional computed var 'changes' to enum and move switch there?